sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -336,8 +337,8 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
       if (ctx.partitionSpec != null) {
         throw new ParseException("DESC TABLE COLUMN for a specific partition is not supported", ctx)
       } else {
-        DescribeColumnCommand(
-          visitTableIdentifier(ctx.tableIdentifier),
```

Comment:
Now that these rules create `DescribeColumnStatement` and `DescribeTableStatement`, they should be moved into Catalyst. There isn't anything specific to the implementation any more.

Reply:
Do you mean that `DescribeColumnCommand` and `DescribeTableCommand` should be moved to Catalyst? The V1 commands depend on a bunch of stuff that's in core, such as `SparkSession` and `DDLUtils`.